policyengine_us/parameters/gov/states/mn/tax/income/subtractions/social_security/income_amount.yaml

```diff
@@ -3,26 +3,42 @@ metadata:
   label: Minnesota social security subtraction income amount
   period: year
   unit: currency-USD
+  uprating:
+    parameter: gov.irs.uprating
+    rounding:
+      type: nearest
+      interval: 10
   breakdown:
   - filing_status
   reference:
+  - title: Minnesota Statutes 290.0132 Subd. 26.Social Security benefits. (f)(g)(h)(j)
+    href: https://www.revisor.mn.gov/statutes/cite/290.0132#stat.290.0132.26
+  - title: 2023 Minnesota Statutes Section 270C.22 — Cost Of Living Adjustment Subdivision 1.e
+    href: https://www.revisor.mn.gov/statutes/cite/270c.22#stat.270C.22.1
   - title: 2021 Minnesota Schedule M1M, Income Additions and Subtractions, Line 12 worksheet
     href: https://www.taxformfinder.org/forms/2021/2021-minnesota-form-m1m.pdf#page=4
   - title: 2022 Minnesota Schedule M1M, Income Additions and Subtractions, Line 12 worksheet
-    href: https://www.revenue.state.mn.us/sites/default/files/2023-01/m1m_22.pdf#page=4
+    href: https://www.revenue.state.mn.us/sites/default/files/2024-01/m1m-22.pdf#page=4
+  - title: 2023 Minnesota Schedule M1M, Income Additions and Subtractions, Line 12 worksheet
+    href: https://www.revenue.state.mn.us/sites/default/files/2024-04/m1m-23_0.pdf#page=4
```

Comment:
We need to uprate the breakdown parameters individually, same as for the single amount parameters, each value needs metadata for it to work, 

Example:
policyengine_us/parameters/gov/states/vt/tax/income/deductions/standard/base.yaml

Reply:
all breakdown params